components/connectivity-adapter/internal/connectorservice/connector/client.go

```diff
@@ -18,15 +17,11 @@ type client struct {
 	timeout       time.Duration
 }
 
-func NewClient(compassConnectorAPIURL string, timeout time.Duration) (Client, error) {
-	gqlExternalAPIClient := graphql.NewClient(compassConnectorAPIURL, graphql.WithHTTPClient(&http.Client{}))
-
-	client := &client{
-		gqlAPIClient: gqlExternalAPIClient,
-		timeout:      timeout,
+func NewClient(gqlClient *graphql.Client, timeout time.Duration) Client {
```

Comment:
timeout argument is no longer used (timeout value hardcoded)

Reply:
That's correct